core/README.md

```diff
@@ -19,4 +19,14 @@ We use [Maven](https://maven.apache.org) for the build. Simply run as below comm
 mvn clean package
 ```
 
-After a successful build, the jar of 'rapids-4-spark-tools_2.12-*-SNAPSHOT.jar' will be in 'target/' directory.
+After a successful build, the jar of 'rapids-4-spark-tools_2.12-*-SNAPSHOT.jar' will be in 'target/' directory.  
+This will build the plugin for a single version of Spark. By default, this is Apache Spark 3.1.1.
+
+To build against other versions of Spark you use the `-Dbuildver=XXX` command line option to Maven.  
+For instance to build Spark 3.3.0 you would use:
```

Comment:
it might be nice to expand this to say this is meant to for dev purposes and to allow running the tests agianst different versions until we split the tests out

Reply:
Done!